### PR TITLE
fix: only confirm selections when clicking confirm or outside

### LINKED ISSUE #86: Selections are confirmed when moving to another page in the agnostic table
Selections are confirmed when moving to another page in the agnostic table. This makes it impossible to make selections on other dimension values then on the ones displayed on the page where we start making selections on.

### PR DESCRIPTION
**Pull request by haxxmaxx**
fixes #86

Since we don't have to cancel clicks all over the chart, we only need one listener on the table body and `selectCell` checks if we are pressing a valid cell

### CODE REVIEW

File: src/table/selections-utils.js [hunk 1]
```diff
@@ -74,10 +74,12 @@ export function selectCell(cell, selState, selDispatch, evt) {
   const { rowIdx, colIdx, qElemNumber } = cell;
   let selectedRows = [];
 
-  if (!api.isActive()) {
+  if (selState.colIdx === -1) {
```
Comment (haxxmaxx) [inline src/table/selections-utils.js:77]: This is always up to date, while there is an bug on isActive (returning false positives)

**Review by LiKang6688 — APPROVED** [review]
looks good to me. tried in the sense-client and did not find any issue.

File: src/table/selections-utils.js (hunk 1 above)
Comment (LiKang6688) [reply src/table/selections-utils.js:77]: Could you help me understand the colIdx?

**Comment by haxxmaxx** [comment]
> looks good to me. tried in the sense-client and did not find any issue.

Sense works fine, But there is a bug running in nebula https://github.com/qlik-oss/nebula.js/issues/576. I'll merge this for now, Tobias is working on the nebula bug

**Pull request merged**